definitions/artifacts/azure-databricks-workspace.json

```diff
@@ -0,0 +1,87 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"$id": "https://schemas.massdriver.cloud/definitions/artifacts/azure-databricks-workspace.json",
+	"$md": {
+		"access": "public"
```

Comment:
@coryodaniel before I merge this, do we want to make this access level different to avoid exposing to more customers than mckinsey for the time being?

Reply:
I dont believe customers will see it. All of the UI stuff we expose is based on opt-in from $md config. Since it doesnt have any of [these](https://github.com/massdriver-cloud/artifact-definitions#md-config) annotations I think its fine to make it public.

That being said... I'd like to see it as private because we haven't tested that any of our link/connector selectors in the UI actually connect private artifacts, so it might be nice to use this to test that and then we can make it public so it doesnt blow up on us in 2 months when we can't figure out why its not connectable to other teams.


@xpositivityx to validate my first paragraph.